spyder/app/mainwindow.py

```diff
@@ -88,77 +83,44 @@
 from qtawesome.iconic_font import FontError
 
 
-#==============================================================================
+# ============================================================================
 # Proper high DPI scaling is available in Qt >= 5.6.0. This attibute must
 # be set before creating the application.
-#==============================================================================
+# ============================================================================

```

Comment:
This block comment doesn't make sense here now. Please remove it.

Reply:
Fixed